src/main/kotlin/uk/gov/justice/digital/hmpps/nomisprisonerapi/jpa/CSIPReport.kt

```diff
@@ -30,14 +30,18 @@ data class CSIPReport(
   @GeneratedValue(generator = "CSIP_ID")
   val id: Long = 0,
 
+  @Column(name = "CSIP_SEQ")
+  val sequence: String? = null,
+
   @ManyToOne(fetch = FetchType.LAZY)
   @JoinColumn(name = "OFFENDER_BOOK_ID")
-  val offenderBooking: OffenderBooking?,
```

Comment:
The db defines the offenderBooking as nullable - however, none of the entries in this table have a missing offenderBooking number.  Also, the trigger on the insert gets the prisonId from the offenderbooking and fails if this is null.
I am therefore not convinced it should be nullable....so have change this to non-nullable (if that makes sense)